src/components/cesium_map/api/spatial.js

```diff
@@ -262,10 +264,10 @@ export class PointStreamPrimitiveCollection extends Cesium.PointPrimitiveCollect
           }
           const p0 = Cesium.Cartesian3.fromDegrees(doc.x, doc.y, 10);
           this.add({
+            id: doc.id,
             position: p0,
             color: this.outlineStyle(locations, location),
-            pixelSize: 10,
-            heightReference: Cesium.HeightReference.RELATIVE_TO_GROUND,
```

Comment:
@datadavev I just review `cesium` documentation. `PointPrimitive` doesn't support `HeightReference`, which means some primitive points will be below the `terrain`.

Reply:
This will be a problem for rendering, especially where the terrain is quite variable. There's two options: Use a billboard or other geometry that can be height relative, or compute elevations for every point. Computing elevations on the fly is very slow, so if we go that route, then elevations will need to be precomputed. I have a tool that does this, we may consider adding this to the indexing workflow (to store a computed Z value with each location), https://github.com/datadavev/elevate